loader/test/prepmod.test.js

```diff
@@ -1,15 +1,28 @@
 const nock = require("nock");
-const { checkAvailability, formatLocation } = require("../src/sources/prepmod");
-const { createSmartLocation } = require("./support/smart-scheduling-links");
-const { expectDatetimeString } = require("./support");
+const config = require("../src/config");
+const {
+  API_PATH,
+  checkAvailability,
+  formatLocation,
+} = require("../src/sources/prepmod");
+const { prepmodHostsByState } = require("../src/sources/prepmod/hosts");
+const {
+  createSmartLocation,
+  createSmartManifest,
+} = require("./support/smart-scheduling-links");
+const { expectDatetimeString, toNdJson } = require("./support");
 const { locationSchema } = require("./support/schemas");
 const { VaccineProduct } = require("../src/model");
 const { EXTENSIONS } = require("../src/smart-scheduling-links");
 
 describe("PrepMod API", () => {
   jest.setTimeout(60000);
 
+  const _apiUrl = config.apiUrl;
+  const _apiKey = config.apiKey;
   afterEach(() => {
+    config.apiUrl = _apiUrl;
+    config.apiKey = _apiKey;
```

Comment:
Don't love this hack. Do we have to mutate these globals?

Reply:
We do, since `checkAvailability` will read the config when it sets up an `ApiClient` instance. I might also be able to use [`jest.mock(moduleName, newModuleDefinition)`](https://jestjs.io/docs/jest-object#jestmockmodulename-factory-options), but IIRC it has significant caveats when mocking properties and not functions. I remember having some issues with it around simple stuff like this before.

Reply:
If you don’t think, say, anything but `cli.js` should ever use the `config.js` module, that’s not crazy, but it does mean we have to refactor everything, which I think is a little out of scope here. 😉

The other option here be just always setting these values to something arbitrary like `"http://univaf.test"` and `"test-api-key"` for all tests, but I was trying to keep it narrow here. We also still need to do this kind of thing in general for other tests that actually test how behavior differs with different configuration, e.g. https://github.com/usdigitalresponse/univaf/blob/fddfe0a71dd187be8bba7e273c5cd59f26f4e769/loader/test/cvs.api.test.js#L22-L39 (although the `config` module never got backported to that source [inherited from New Jersey], it's the same principle).

Reply:
I don't mind at all having one big override for all of the tests, if that would clean up this PR. But I don't think it's too big of a deal either way. Just struck me as odd.